examples/test.h

```diff
@@ -0,0 +1,17 @@
+#ifndef TEST_H
+#define TEST_H
+#include <unistd.h>
+#include <assert.h>
+
+#define SUCCESS_MARKER "<<success>>\n"
+
+int test_success()
+{
+    // below unchecked syscall return code
+    write(STDERR_FILENO, SUCCESS_MARKER, sizeof(SUCCESS_MARKER));
+    return 0;
+}
+
+#define test_assert(expr) assert(expr && "<<error>>")
```

Comment:
I don't undestand how it is supposed to work. Why _logical and_ over `expr` and a string literal??

Reply:
It's based on the following post:
https://stackoverflow.com/questions/3692954/add-custom-messages-in-assert